book.md

```diff
@@ -1332,7 +1332,7 @@ will include bounds checks.
 Successfully using bounds checking compiler features for a large codebase
 requires substantial effort. An example of this is refactoring the Linux kernel
 to use bounds checks for flexible arrays, as described in [Kees Cook's
-blog]([@Cook2023]).
+blog](@Cook2023).
```

Comment:
```suggestion
blog](https://people.kernel.org/kees/bounded-flexible-arrays-in-c).
```

Reply:
Thank you for reporting this @kees !
When I tested locally, just removing the square brackets still resulted in a broken link.
Ultimately, I think the root issue here is mixing citation syntax (starting with `@` syntax) and the markdown syntax to create a regular link.
For this reference, I thought that probably it would be best to have a regular link, as the target is a blog post, rather than a formal citation. Not going with a citation also means that the reader gets taken to your blog post directly rather than having an indirection through the bibliography and having to click a second time to get to the blog post.

Does that seem allright to you?

Reply:
I was just trying to follow the examples in the rest of the doc. For example, the bib linkage for Aleph's stuff seems to work as expected. It seems without the linkage, the array-bounds blog post doesn't end up appearing at the end of the document either, so I think we should fix it to show up. I noticed others have comments like "leave this with a trailing comma" but I didn't understand what that was about. Is that needed here for some reason?

Reply:
I'm not sure where those "leave this with a trailing comma" comments come from in the book.bib file. Maybe @g-kouv knows? (Although don't expect an answer soon, I expect Georgia to be offline for a while).

I'm happy either way with having an entry in the bibliography or linking directly. Thanks for fixing this!
I'll be merging this soon.

Reply:
I experimented with this: a trailing comma means "do not order by surname". So I have no trailing comma and correctly appear as "Cook, Kees" but Aleph One does, and correctly appears as "Aleph One" in the references.